Userland/Libraries/LibGL/Vertex.cpp

```diff
@@ -117,7 +117,12 @@ void GLContext::gl_color_pointer(GLint size, GLenum type, GLsizei stride, void c
         GL_INVALID_ENUM);
     RETURN_WITH_ERROR_IF(stride < 0, GL_INVALID_VALUE);
 
-    m_client_color_pointer = { .size = size, .type = type, .normalize = true, .stride = stride, .pointer = pointer };
+    if (m_array_buffer) {
+        size_t data_offset = (size_t)pointer;
```

Comment:
Please no C-style casts, so change to `static_cast<size_t>`. Multiple instances of this in this unit.

Reply:
I used `reinterpret_cast` since C++ doesn't allow static casts from `void*` to `size_t`.

Reply:
Oof, repurposing `const void*` as a byte offset is very :yakid2:, your new solution seems fine